$NetBSD: patch-ab,v 1.3 2013/08/07 03:11:35 mef Exp $

See Makefile for following part. Honor ${TOOLS_PATH.sed}
SUBST_CLASSES+=		sed

--- fixincludes/fixincl.x.orig	2009-08-27 00:05:13 +0300
+++ fixincludes/fixincl.x	2009-08-27 00:05:32 +0300
@@ -36,9 +36,7 @@
  *  You should have received a copy of the GNU General Public License along
  *  with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-#ifndef SED_PROGRAM
-#define SED_PROGRAM "/usr/bin/sed"
-#endif
+#define SED_PROGRAM "@SED@"
 static char const sed_cmd_z[] = SED_PROGRAM;
 
 /* * * * * * * * * * * * * * * * * * * * * * * * * *
